projects/plugins/boost/app/lib/critical-css/source-providers/providers/Singular_Post_Provider.php

```diff
@@ -105,12 +105,29 @@ public static function describe_key( $provider_key ) { // phpcs:ignore Generic.C
 	 * @return mixed|void
 	 */
 	public static function get_post_types() {
-		$post_types = get_post_types( array( 'public' => true ) );
+		$post_types = get_post_types(
+			array(
+				'public'              => true,
+				'exclude_from_search' => false, // Only return post types that are searchable.
```

Comment:
Why only searchable post types?

Reply:
Hm, I don't recall what my reasoning for this was, but thanks for bringing it up.

It doesn't make sense to ignore post types that aren't searchable, as that might be intended for a custom post type that is public.

I've reverted this change.